wagtail_wordpress_import/prefilters/transform_styles_filter.py

```diff
@@ -0,0 +1,221 @@
+import re
+
+from bs4 import BeautifulSoup
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+
+def normalize_style_attrs(html, options=None):
```

Comment:
**Comment**: It's a bit clunky that this function takes a string, parses it with Beautifulsoup, and returns a string, only for the calling code at https://github.com/torchbox/wagtail-wordpress-import/pull/48/files#diff-ba7adf08e8b8a91567d4cac5f2059c5cfee02371cfc8aad3b80d29a1046d931aR55 to parse the output a second time. Would it be possible for this function to take and return a soup object?